restApi.py

```diff
@@ -44,5 +44,16 @@ def get_watched_rooms():
    rows = c.fetchall()
    return json.dumps(results)    
 
+@app.route('/delete-watch/<int:watch>')
+def delete_watched_rooms(watch):
+   conn = sqlite3.connect("BOOKING.db")
+   conn.row_factory = sqlite3.Row
+   results = []
+   
+   c = conn.cursor()
+   c.execute("DELETE FROM WATCHED WHERE WatchedId=%s" % watch)
+   conn.commit()
+   conn.close()
+
```

Comment:
I think we need a return statement for each endpoint.
After a quick test I'm getting:
``` ERROR in app: Exception on /delete-watch/19 [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python3.6/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
```
This did in fact delete the record from the database but it probably doesn't look good for us to have a 500 response.
Maybe the return value could be something like success or failure depending on whether the database call succeeded.